build/android/aab_to_apk.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import argparse
+import os
+import shlex
+import subprocess
+import sys
+import zipfile
+import sign_apk
+
+
+def main():
+    argument_parser = argparse.ArgumentParser()
+    argument_parser.add_argument('bundletool')
+    argument_parser.add_argument('target_aab_path')
+    argument_parser.add_argument('output_apk_path')
+    argument_parser.add_argument('output_path')
+    argument_parser.add_argument('key_path')
+    argument_parser.add_argument('key_passwd')
+    argument_parser.add_argument('prvt_key_passwd')
+    argument_parser.add_argument('key_name')
+    argument_parser.add_argument('zipalign_path')
+    argument_parser.add_argument('apksigner_path')
+    argument_parser.add_argument('jarsigner_path')
+    args = argument_parser.parse_args()
+
+    apks_name = os.path.splitext(args.output_apk_path)[0] + ".apks"
+
+    if os.path.isfile(apks_name):
+        os.remove(apks_name)
+    command_line = "java -jar " + args.bundletool + " build-apks " + \
+                   "--bundle=" + args.target_aab_path + \
+                   " --output=" + apks_name + \
+                   " --mode=universal " + \
+                   "--ks=" + args.key_path + \
```

Comment:
maybe here also the 1st space is required " --ks=" ? and for " --bundle=" also?

Reply:
there is a space after `--mode=universal`

Reply:
indeed, I missed that, thanks